pkg/resources/stream.go

```diff
@@ -152,13 +152,13 @@ func streamOnTableIDFromString(stringID string) (*streamOnTableID, error) {
 }
 
 // CreateStream implements schema.CreateFunc
-func CreateStream(d *schema.ResourceData, meta interface{}) error {
+func CreateStream(data *schema.ResourceData, meta interface{}) error {
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>R014: *schema.ResourceData parameter of CreateFunc, ReadFunc, UpdateFunc, or DeleteFunc should be named d